src/overlays/actors/ovl_Boss_02/z_boss_02.c

```diff
@@ -97,186 +151,2146 @@ const ActorInit Boss_02_InitVars = {
     (ActorFunc)Boss02_Draw,
 };
 
-// static ColliderJntSphElementInit sJntSphElementsInit[22] = {
-static ColliderJntSphElementInit D_809DF614[22] = {
+static f32 D_809DF5B0 = 1.0f;
+
+static s16 D_809DF5B4[] = {
+    0x0000, 0x00C3, 0x00BE, 0x00B9, 0x00B4, 0x00AF, 0x00AA, 0x00A5, 0x00A0, 0x009B, 0x0096, 0x0091,
+    0x008C, 0x0087, 0x0082, 0x007D, 0x0078, 0x0073, 0x006E, 0x0069, 0x0064, 0x005F, 0x005A, 0x0000,
+};
+
+static s16 D_809DF5E4[] = {
+    0x0000, 0x00C4, 0x00C0, 0x00BC, 0x00B8, 0x00B4, 0x00B0, 0x00AC, 0x00A8, 0x00A4, 0x00A0, 0x009C,
+    0x0098, 0x0094, 0x0090, 0x008C, 0x0088, 0x0084, 0x0080, 0x007C, 0x0078, 0x0074, 0x0070,
+};
```

Comment:
```suggestion
static s16 D_809DF5B4[] = {
    0, 195, 190, 185, 180, 175, 170, 165, 160, 155, 150, 145,
    140, 135, 130, 125, 120, 115, 110, 105, 100, 95, 90, 0,
};

static s16 D_809DF5E4[] = {
    0, 196, 192, 188, 184, 180, 176, 172, 168, 164, 160, 156,
    152, 148, 144, 140, 136, 132, 128, 124, 120, 116, 112,
};
```
These are used as indices, so they can be decimal